core/src/main/java/nl/nn/adapterframework/logging/IbisLoggerConfigurationFactory.java

```diff
@@ -140,6 +157,7 @@ public static String readLog4jConfiguration(InputStream stream) throws IOExcepti
 				if(checkVersionOnlyFirst1024Characters) {
 					//See if log4j2 prefix is somewhere in the first 1024 characters
 					if(!stringWriter.toString().contains("<log4j2:Configuration")) {
+						System.err.println(LOG_PREFIX + "detected obsolete configuration format. Please use the log4j2 layout in file log4j4ibis.xml");
 						throw new IllegalStateException("Detected obsolete configuration format. Please use the log4j2 layout in file log4j4ibis.xml");
```

Comment:
Je weet niet of het obsolete is, je hebt alleen maar geen "<log4j2:Configuration" gezien.
Ik denk dat je moet zeggen: "did not recognize configuration format, expected log4j2 layout.
Wordt het vervolgens nog wel geïnterpreteerd? dan dat ook zeggen.